### PR TITLE
Remove duplication

### PR DESCRIPTION
**Pull request by mooncak**
Signed-off-by: mooncake <xcoder@tenxcloud.com>

Remove duplication in files:

1. mentoring/group-mentee-guide.md
2. sig-scalability/processes/formal-scalability-processes.md

### CODE REVIEW

**Comment by nikhita** [comment]
I used [`pcregrep`](https://www.pcre.org/original/doc/html/pcregrep.html) in a quick and dirty bash script to find all occurrences of duplicate words, which essentially does:

```
pcregrep -Min --color=auto '\b([^[:space:]]+)[[:space:]]+\1\b' $i
```

This generates lots of false positives. I manually went through the results and rejected the wrong ones. I did not include the election bios.

This is the complete list of duplicate words:

```
./events/2017/05-leadership-summit/announcement.md
3:2016, but involving a smaller smaller audience comprised solely of leaders and influencers of the community. These leaders and

./events/2016/fixit201606.md
7:Another important piece of fixit culture is rewards.  **Everyone who contributes (as measured by engagements on Github) between 6/27 and 7/1 will receive Kubernetes stickers and all merged PRs that are contributed during the fixit will will receive a Kubernetes embroidered patch.**

./mentoring/group-mentee-guide.md
26:* Effective communication in our our ecosystem

./sig-scalability/processes/formal-scalability-processes.md
72:This phase constitutes the final layer of protection against regressions before cutting the release. We already have scalability CI jobs in place for this. The spectrum of scale they cover is quite wide, ranging from 100-node to 5000-node clusters (both for kubemark and real clusters). However, what what we need additionally is:

./contributors/design-proposals/multicluster/federated-replicasets.md
350:   a cluster is lost completely then the cluster is removed from the
   the cluster list (or marked accordingly) so
385:failed to be scheduled. Cluster capacity is the difference between the
the placed and failed to be scheduled.

./communication/meeting-notes-archive/q1-2_2018_community_meeting_minutes.md
5:153:  - This doesn't solve our Slack lock-in concerns, but can be a good first step in being more active in running our own community properties so that we can build out own own resources.
8:141:* When `kubectl` is executed with a subcommand _foo_ that does not exist exist in the command tree, it will attempt to look

./sig-cli/migrated-from-wiki/roadmap-kubectl.md
16:  * [Default manifest manifest](https://github.com/kubernetes/kubernetes/issues/3268) (ala Dockerfile or Makefile)

./keps/sig-node/0030-20180906-quotas-for-ephemeral-storage.md
522:  the problem.  Against the possibility that that cannot be done

./keps/sig-contributor-experience/0007-20180403-community-forum.md
153:  - This doesn't solve our Slack lock-in concerns, but can be a good first step in being more active in running our own community properties so that we can build out own own resources.

./keps/sig-cli/0024-kubectl-plugins.md
141:* When `kubectl` is executed with a subcommand _foo_ that does not exist exist in the command tree, it will attempt to look
```

The first line is the file name. The first character of the second line is the line number and then it's the line containing the duplicate words.

@mooncak Can you correct all of these in this PR? That would truly appreciated. Thanks!

**Comment by mooncak** [comment]
@nikhita  already correct it.

File: events/2016/fixit201606.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Google runs internal fixit weeks. During these weeks the teams set aside all  cr
...
 
-Another important piece of fixit culture is rewards.  **Everyone who contributes (as measured by engagements on Github) between 6/27 and 7/1 will receive Kubernetes stickers and all merged PRs that are contributed during the fixit will will receive a Kubernetes embroidered patch.**
+Another important piece of fixit culture is rewards.  **Everyone who contributes (as measured by engagements on Github) between 6/27 and 7/1 will receive Kubernetes stickers and all merged PRs that are contributed during the fixit will receive a Kubernetes embroidered patch.**
```
Comment (nikhita) [inline events/2016/fixit201606.md]: Looks like "historical" events don't need to be updated. Can you remove this change and squash the commits? Sorry for not mentioning this earlier.

Comment (mooncak) [reply events/2016/fixit201606.md]: Done

**Comment by nikhita** [comment]
@mooncak Thanks for addressing feedback so quickly! :)

There are some more duplicate words in the list above. Can you fix them too? I'm listing the ones that are present in https://github.com/kubernetes/community/pull/2898#issuecomment-435686111 but not in the PR so that it's easier for you to fix it.

It looks great as a whole. Will add the lgtm once these changes are added. :tada: 

```
./contributors/design-proposals/multicluster/federated-replicasets.md
350:   a cluster is lost completely then the cluster is removed from the
   the cluster list (or marked accordingly) so
385:failed to be scheduled. Cluster capacity is the difference between the
the placed and failed to be scheduled.

./communication/meeting-notes-archive/q1-2_2018_community_meeting_minutes.md
5:153:  - This doesn't solve our Slack lock-in concerns, but can be a good first step in being more active in running our own community properties so that we can build out own own resources.
8:141:* When `kubectl` is executed with a subcommand _foo_ that does not exist exist in the command tree, it will attempt to look

**Comment by mooncak** [comment]
@nikhita  I don't see any duplicate in ./communication/meeting-notes-archive/q1-2_2018_community_meeting_minutes.md
can you double check your local files is the newest?

**Comment by nikhita** [comment]
@mooncak My bad. :) I think I copy-pasted it wrong while manually checking the results. Those lines were in some other file and they are already fixed.

I think the only file remaining is `./contributors/design-proposals/multicluster/federated-replicasets.md`.

**Comment by mooncak** [comment]
> @mooncak My bad. :) I think I copy-pasted it wrong while manually checking the results. Those lines were in some other file and they are already fixed.
> 
> I think the only file remaining is `./contributors/design-proposals/multicluster/federated-replicasets.md`.

Fixed

**Comment by nikhita** [comment]
@mooncak This looks great! Thanks for bearing through the back and forth. :)

/lgtm

/assign @cblecker 
for approval and for adding the `tide/squash` label

**Pull request merged**